package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "itapuc-apprenticeship-project",
+  "version": "1.0.0",
+  "description": "Task 2",
```

Comment:
This field should contain a description of the project, not this task.  The file describes the package we are building - the backend for our chat application.